changelog.txt

```diff
@@ -19,12 +19,15 @@ that repo.
 - `warn-stealers`: warn when creatures that may steal your food, drinks, or items become visible
 
 ## Fixes
+- `devel/query`: updates call to deleted function that was causing an error for tile data
```

Comment:
I'm not sure this is understandable by readers of the changelog. The fact that the error was caused by calling a deleted function is not useful to tell users. Could you rephrase this as something like "fixed error when [use case]"?

Reply:
In other words, try to answer the question for the user: "What can I do now that I couldn't do before?"

Reply:
It is a development script, does that matter here?

Reply:
As a side note, I don't precisely know all of what this means for using the script. I just found what was causing the error and fixed it, it may have other edge cases than just `--tile`

Reply:
I think that's ok if it fixes other things too. Rephrasing the message as "fixed error when --tile is specified" is definitely good enough.

Reply:
```suggestion
- `devel/query`: fixed error when --tile is specified
```